src/Esfa.Vacancy.Register.Api/Global.asax.cs

```diff
@@ -25,5 +25,26 @@ protected void Application_PreSendRequestHeaders(object sender, EventArgs e)
         {
             HttpContext.Current?.Response.Headers.Remove("Server");
         }
+
+        protected void Application_Error(object sender, EventArgs e)
+        {
+            var ex = Server.GetLastError();
+
+            if (ex is HttpException httpError)
+            {
+                ApiRedirect("~/api/error");
+                return;
+            }
+
+            var logger = DependencyResolver.Current.GetService<ILog>();
+            if (ex != null) logger.Warn(ex, ex.Message);
+        }
+
+        private void ApiRedirect(string path)
+        {
+            Response.Clear();
```

Comment:
👍

Reply:
You could actually narrow down on the status codes that you want to forward to error controller. So for instance you may not want to forward 500 to error controller.

Reply:
Are we not just intercepting work the VacancyApiExceptionHandler would already do? Do we need to handle errors in global.asax?